lib/workers/repository/master-issue.ts

```diff
@@ -22,12 +20,16 @@ function getListItem(branch, type, pr) {
   return item + ' (' + uniquePackages.join(', ') + ')\n';
 }
 
-async function ensureMasterIssue(config, branches) {
+export async function ensureMasterIssue(
+  config: RenovateConfig,
+  branches: BranchConfig[]
+): Promise<void> {
   if (
     !(
       config.masterIssue ||
       branches.some(
-        branch => branch.masterIssueApproval || branch.masterIssuePrApproval
+        (branch: BranchConfig) =>
```

Comment:
I think this is not necessary, because `branches` has a concrete type.

Others below too.